### DISCUSSION TITLE
Problems on the installation of MOSEK on Mac Catalina

### DISCUSSION THREAD

**Discussion by Googoogaga558** [cvxpy/cvxpy#2045] [Q&A]
I don't know which step is wrong that i can't even run "msktestlic" on the Terminal. I'm using the macos catalina and the date of license is later than the date of version. The version of MOSEK is 10.0

First I downloaded the MOSEK and unpack it under "/Users/myname", /Users/myname/mosek

then I run the installation 

Next, run into"vi ~/.zshell.profile" to export the path

License is also put into the "/Users/myname/mosek"

i also added all the subfolders of mosek to MATLAB, then i reopened the MATLAB, but it shows that

>> mosekdiag
Unrecognized function or variable 'mosekdiag'.

And the msktestlic is also not working on Terminal.
 "zsh: command not found: msktestlic"

Hope to know the wrong site.

BW

**Comment by MyeshaSteltzerZT1** [comment]
Content Brief: Troubleshooting MOSEK Installation on macOS Catalina

Keywords:

MOSEK
macOS Catalina
license
installation
Terminal
MATLAB
msktestlic
mosekdiag
Header Outline:
I. Introduction

Briefly introduce the problem and the context of the issue
II. Steps Taken to Install MOSEK

Explain the steps taken to install MOSEK on macOS Catalina
Emphasize the importance of following the installation instructions carefully
III. Error Messages

Explain the error messages received when attempting to run mosekdiag and msktestlic
Provide potential reasons why these commands are not working
IV. Troubleshooting Steps

Provide a list of troubleshooting steps to identify the issue and resolve it
Include specific commands and terminal instructions to execute each step
V. Conclusion

Recap the problem and the steps taken to resolve it
Offer additional resources for further assistance
Title Tags:

Troubleshooting MOSEK Installation on macOS Catalina: How to Fix msktestlic and mosekdiag Errors
Solving msktestlic and mosekdiag Errors on macOS Catalina: A Step-by-Step Guide
MOSEK Installation Issues on macOS Catalina: Resolving msktestlic and mosekdiag Errors
Meta Descriptions:

Having trouble running msktestlic or mosekdiag after installing MOSEK on macOS Catalina? Check out this step-by-step guide to troubleshoot and fix the issue.
If you're getting error messages when trying to run msktestlic or mosekdiag on macOS Catalina, this guide will help you identify and resolve the problem.
Don't let msktestlic and mosekdiag errors stop you from using MOSEK on your macOS Catalina. Follow these troubleshooting steps to get back on track.

**Reply by phschiele** [reply]
@MyeshaSteltzerZT1 Installation issues with solvers are best addressed directly via their respective channels. I hope you found a way to resolve this issue by now.
